### PR TITLE
W-10984948 - Added Long datatype

### PR DESCRIPTION
**Pull request by looseale**

### CODE REVIEW

**Comment by uip-robot-zz** [comment]
Git2Gus App is installed but the `.git2gus/config.json` doesn't exist.

File: shared/src/main/scala/amf/core/internal/plugins/document/graph/parser/GraphParserHelpers.scala [hunk 1]
```diff
@@ -53,6 +52,7 @@ trait GraphParserHelpers extends GraphContextHelper {
     detectedType match {
       case DataType.Boolean => bool(node)
       case DataType.Integer => int(node)
+      case DataType.Long    => double(node)
```
Comment (nschejtman) [inline shared/src/main/scala/amf/core/internal/plugins/document/graph/parser/GraphParserHelpers.scala]: Shouldn't this be `int(node)`?

Comment (nschejtman) [reply shared/src/main/scala/amf/core/internal/plugins/document/graph/parser/GraphParserHelpers.scala]: or `long(node)`

Comment (looseale) [reply shared/src/main/scala/amf/core/internal/plugins/document/graph/parser/GraphParserHelpers.scala]: Yep, it was an error. Fixed

**Review by arielmirra — APPROVED** [review]

**Review by nschejtman — APPROVED** [review]

**Pull request merged**